Hexastack/Hexabot
# feat/refactor helpers

# Motivation

In this PR we refactor helper so that it will be similar to channels and plugins : 
- Added "helper" module
- Refactored and renamed the default nlp helper
- Add default NLU helper in settings
- Add helper i18n 

# How to test
- NLU prediction should work as usual
- Chat flow should work with NLU triggers as usual
- Ability to Import and export samples should work as usual
- New setting "Default NLU Helper" should appear in the Chatbot Settings
- NLU Settings Tab should appear as usual (with no provider)

# Type of change:

Please delete options that are not relevant.


# Checklist:
